doc/howtos/rdtraining/newapp.rst

```diff
@@ -0,0 +1,103 @@
+.. _howto/rdtraining/newapp:
+
+=================
+A New Application
+=================
+
+The purpose of this chapter is to lay the basis for the creation of a completely new Odoo module.
+We will start from scratch with the minimum necessary to have our module recognized by Odoo.
+During the next chapters, we will progressively add features to build a realistic business case.
+
+The Real Estate Advertisement module
+====================================
+
+Our new module will cover a business area which is very specific, and therefore not included in the
+standard set of modules: the real estate properties. It is worth noting that before
+developing a new module, it always makes sense to verify if Odoo doesn't already provide a way
+to answer the specific business needs.
+
+Here is an overview of the main list view containing some advertisements:
+
+.. image:: newapp/media/overview_list_view_01.png
+   :align: center
+   :alt: List view 01
+
+The top area of the form view summarizes important information for the property, such as the name,
+the property type, the postcode and so on. The first tab contains information describing the 
+property: bedrooms, living area, garage, garden...
+
+.. image:: newapp/media/overview_form_view_01.png
+   :align: center
+   :alt: Form view 01
+
+The second tab lists off the offers for the property. Indeed, potential buyers may offer more or
+less than the expected selling price. It is up to the seller to accept an offer.
+
+.. image:: newapp/media/overview_form_view_02.png
+   :align: center
+   :alt: Form view 02
+
+Here is a quick video showing the workflow of the module.
+
+TODO: video showing features
+
+Prepare the addon directory
+===========================
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`manifest <reference/module/manifest>`.
+
+.. note::
+
+   **Goal**: the goal of this section is to have Odoo recognize our new module, which will still
+   be an empty shell. It will be listed in the Apps:
+
+   .. image:: newapp/media/app_in_list.png
+      :align: center
+      :alt: The new module appears in the list
+
+The very first step of a module creation is to create a new folder. To ease the development, we
+suggest you first create the folder ``/home/<user>/src/custom``. In this folder we add the folder
```

Comment:
```suggestion
suggest you first create the folder ``/home/$USER/src/custom``. In this folder we add the folder
```
maybe we can change <user> to $USER to be coherent with the setup.rst file (or vise versa, change in the setup.rst file the $USER to <user>